### ISSUE TITLE
Entire package import error [ moved method ]

### ISSUE THREAD

**Issue by MOHAND99** [metal3d/keras-video-generators#70]
**System Information**
- your operating system: 
- your python version (`python -version`):
- keras video generator version (`pip freeze | grep keras-video-generators`): 

**Describe the bug**
A clear and concise description of what the bug is. Please provide the the error output with the traceback and if possible a little code snippet to reproduce the error)
ImportError                               Traceback (most recent call last)
[<ipython-input-9-ea2ca17e3187>](https://localhost:8080/#) in <module>
----> 1 import keras_video

2 frames
[/usr/local/lib/python3.7/dist-packages/keras_video/__init__.py](https://localhost:8080/#) in <module>
     31 __version__ = "1.0.14"
     32 
---> 33 from . import flow
     34 from . import generator
     35 from . import sliding

[/usr/local/lib/python3.7/dist-packages/keras_video/flow.py](https://localhost:8080/#) in <module>
     19 import numpy as np
     20 import cv2 as cv
---> 21 from .generator import VideoFrameGenerator
     22 import keras.preprocessing.image as kimage
     23 

[/usr/local/lib/python3.7/dist-packages/keras_video/generator.py](https://localhost:8080/#) in <module>
     17 
     18 from keras.utils import Sequence
---> 19 from keras.preprocessing.image import \
     20     ImageDataGenerator, img_to_array
     21 

ImportError: cannot import name 'img_to_array' from 'keras.preprocessing.image' (/usr/local/lib/python3.7/dist-packages/keras/preprocessing/image.py)

**Comment by dq04171877** [comment]
I have the same problem on colab
<img width="1329" alt="image" src="https://user-images.githubusercontent.com/17245106/197812577-049ef8ee-7ccb-4c8b-bb3a-889fde25bad6.png">

**Comment by adolfojara10** [comment]
This worked for me using the Google collab. After printing out the tensorflow version and seein git is 2.5 I opened /usr/local/lib/python3.7/dist-packages/keras_video/generator.py
I replaced line 19: from keras.preprocessing. image import img_to_array 
with: from tensorflow.keras.preprocessing. image import ImageDataGenerator, img_to_array 
saved it and it now works for me

**Comment by MOHAND99** [comment]
i needed to change it for every session

**Comment by adolfojara10** [comment]
Yeah, it is quite uncomfortable to do it every session, but I'm getting used to it. I hope someone else knows how to solve it more efficiently
